doc/OVERMAP.md

```diff
@@ -271,6 +322,15 @@ locations may instead be specified for the entire special using the top level `l
 value for an individual entry takes precedence over the top level value, so you may define the top
 level value and then only specify it for individual entries that differ.
 
+### City distance and size 
+
+During generation of a new overmap, cities and their connecting roads will be generated before 
+specials are placed. Each city gets assigned a size at generation and will begin its life as a single 
+intersection. The city distance field specifies the minimum and maximum distance the special can be 
+placed from this intersection, *not* from the edge of the city, meaning a special with a low minimum
+distance and a relaxed city size restriction may be placed on the outer border of a larger city.
```

Comment:
What is meant by a "relaxed city size restriction", and how does the city size field work, BTW? Only counts cities in that range for distance, or what?

Reply:
If the city size maximum is not smaller than the city distance minimum. All specials are considered to belong only to the nearest city, using the central intersection of that city as their origin.